src/cli/index.ts

```diff
@@ -16,25 +17,34 @@ if (command === 'dev') {
       logTypeScriptMessage('Compiling TypeScript sources...')
     },
     onFileChanged() {
-      logLineWithIcon('⟳ ', 'File change detected. Starting incremental compilation...')
+      logLineWithIcon(
+        '⟳ ',
+        'File change detected. Starting incremental compilation...'
+      )
     },
     onCompilationFailed(message) {
       killDevChildProcess()
       logTypeScriptMessage(message)
     },
     onCompilationDone() {
       killDevChildProcess()
-      devChildProcess = spawn('node', ['./node_modules/@asyncapi/glee/dist/cli/start.js'], {
-        stdio: 'inherit',
-        env: {
-          ...process.env,
-          NODE_ENV: 'development',
-        },
-      })
-    }
+      devChildProcess = spawn(
+        'node',
+        ['./node_modules/@asyncapi/glee/dist/cli/start.js'],
+        {
+          stdio: 'inherit',
+          env: {
+            ...process.env,
+            NODE_ENV: 'development',
+          },
+        }
+      )
+    },
   })
 } else if (command === 'start') {
   import('./start.js')
+} else if (command === 'docs') {
+  docs()
 } else {
```

Comment:
I am unable to create docs, some documentation or example would be awesome. I am using `glee docs` and `glee start` but nothing works I think I need to add something in the `glee.config.js` which I don't find any documentation of.